packages/amplify-graphql-model-transformer/rds-lambda/handler.ts

```diff
@@ -144,6 +144,36 @@ const getSecretManagerValue = async (secretArn: string | undefined): Promise<{ u
   }
 }
 
+const resolveConnectionStringValue = async (jsonConnectionString: string): Promise<string> => {
+  const parsedJsonConnectionString = JSON.parse(jsonConnectionString);
+  const ssmRequestError = 'Unable to connect to the database. Check the logs for more details.';
+  const ssmLoggedError = 'Unable to fetch the connection Uri from SSM for the provided paths.';
+  console.log(`connection uri SSM paths: ${parsedJsonConnectionString}`);
+  if (Array.isArray(parsedJsonConnectionString)) {
+    for (const connectionUriSsmPath of parsedJsonConnectionString) {
+      console.log(`Trying to fetch connection string from SSM path: ${connectionUriSsmPath}`);
```

Comment:
All the console logs - let's print only if debug mode is enabled.

Reply:
This is for debugging E2E, I will remove this in next commit.